src/data/analyse_en.json

```diff
@@ -7,7 +7,7 @@
     "info": "How regularly is the CWA dashboard data updated?",
     "updateNoticeIcon": {
         "title": "Update of the data",
-        "content": "The CWA dashboard data is usually updated daily at 10:30 AM (CET) for the previous day. Data regarding downloads is updated only on working days. On Mondays, the data for Friday, Saturday and Sunday are updated."
+        "content": "<p>The CWA dashboard data is usually updated daily at 10:30 AM (CET) for the previous day. Data regarding downloads is updated only on working days. On Mondays, the data for Friday, Saturday and Sunday are updated.</p>"
```

Comment:
I think this (minor) change should not be in this PR.

Reply:
Hi,

this branch was not created from master. So this branch contains some changes from a previous pull request.
I made a new pull request and will close this request.

Kind regards,

Crystalskulls